test/error.cc

```diff
@@ -270,7 +359,15 @@ Object InitError(Env env) {
       Function::New(env, LastExceptionErrorCode);
   exports["throwJSError"] = Function::New(env, ThrowJSError);
   exports["throwTypeError"] = Function::New(env, ThrowTypeError);
+  exports["throwTypeErrorCtor"] = Function::New(env, ThrowTypeErrorCtor);
+  exports["throwTypeErrorCStr"] = Function::New(env, ThrowTypeErrorCStr);
+  exports["throwTypeErrorNapiVal"] = Function::New(env, ThrowTypeErrorNapiVal);
```

Comment:
Seems like `*NapiVal` methods are not called in the test.

Reply:
Yep. Took these out as that overload was tested by https://github.com/nodejs/node-addon-api/pull/1280/files#diff-ac24d807b22ae295c2dcf786090d933b62aa2b6b3c85c16e839d31ad35fc2c13R81